pkg/virtual/framework/dynamic/apiserver/discovery.go

```diff
@@ -330,3 +332,86 @@ func sortGroupDiscoveryByKubeAwareVersion(gd []metav1.GroupVersionForDiscovery)
 		return version.CompareKubeAwareVersionStrings(gd[i].Version, gd[j].Version) > 0
 	})
 }
+
+type openAPIHandler struct {
```

Comment:
can you describe on the high level how this handler gets a v3 schema for the right resources?

Reply:
go doc I mean